kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/schemaregistry/SchemaRegistryAwareRecordSerDe.java

```diff
@@ -134,13 +134,8 @@ private void fillDeserializedKvBuilder(ConsumerRecord<Bytes, Bytes> rec,
     }
 
     // fallback
-    if (isKey) {
-      builder.key(FALLBACK_FORMATTER.format(rec.topic(), isKey ? rec.key().get() : rec.value().get()));
```

Comment:
lost key method calls?

Reply:
No, isKey ? rec.key().get() : rec.value().get()) was always true for isKey section